[freerdp] Unable to build

```
[ 38%] Linking C shared library libwinpr2.so
[ 38%] Built target winpr
make: *** [Makefile:152: all] Error 2
freerdp failed to build: `make V=0 -j2` exited with 2
```

This would have been a much better error log: 
```
warning: failed to load external entity "/usr/local/share/doc/docbook-xsl-1.79.2/manpages/docbook.xsl"
cannot parse /usr/local/share/doc/docbook-xsl-1.79.2/manpages/docbook.xsl
make[2]: *** [client/X11/CMakeFiles/xfreerdp.manpage.dir/build.make:69: client/X11/xfreerdp.1] Error 4
make[1]: *** [CMakeFiles/Makefile2:4707: client/X11/CMakeFiles/xfreerdp.manpage.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/packages/freerdp.rb
+++ b/packages/freerdp.rb
@@ -2,29 +2,51 @@ require 'package'
 
 class Freerdp < Package
   description 'FreeRDP is a free implementation of the Remote Desktop Protocol.'
-  homepage 'http://www.freerdp.com/'
-  version '2.0.0-rc4'
+  homepage 'https://www.freerdp.com/'
+  version '2.0.0-rc4-1'
   source_url 'https://github.com/FreeRDP/FreeRDP/archive/2.0.0-rc4.tar.gz'
   source_sha256 '3406f3bfab63f81c1533029a5bf73949ff60f22f6e155c5a08005b8b8afe6d49'
 
-  depends_on 'ffmpeg'
-  depends_on 'gst_plugins_base'
   depends_on 'cups'
+  depends_on 'mesa'
   depends_on 'xmlto'
-  depends_on 'libxslt'
   depends_on 'ninja'
   depends_on 'xprop'
-  depends_on 'sommelier'
+  depends_on 'ffmpeg'
+  depends_on 'linux_pam'
+  depends_on 'pulseaudio'
+  depends_on 'libmp3lame'
+  depends_on 'gst_plugins_base'
 
   def self.build
-    system 'cmake', "-DCMAKE_INSTALL_PREFIX:PATH=#{CREW_PREFIX}", '-DWITH_SERVER=yes', '-DWITH_CLIENT=yes', '-DWITH_CHANNELS=yes',
-      '-DBUILD_SHARED_LIBS=yes', '-DWITH_GSSAPI=yes', '-DWITH_DSP_FFMPEG=yes', '-DWITH_LIBSYSTEMD=OFF', '-DWITH_CUPS=ON',
-      '-DWITH_PULSE=ON', "-DDOCBOOKXSL_DIR=#{CREW_PREFIX}/share/doc/docbook-xsl-1.79.2", '-GUnix Makefiles', '.'
-    system 'make'
+    FileUtils.mkdir('build')
+    Dir.chdir('build') do
+      ENV['GSS_ROOT_FLAVOUR'] = 'MIT'
+      system 'cmake',
+             "-DDOCBOOKXSL_DIR=#{CREW_PREFIX}/share/xml/xsl-stylesheets-1.79.2",
+             "-DCMAKE_INSTALL_PREFIX:PATH=#{CREW_PREFIX}",
+             '-DBUILD_SHARED_LIBS=ON',
+             '-DWITH_LIBSYSTEMD=OFF',
+             '-DWITH_DSP_FFMPEG=yes',
+             '-DWITH_CHANNELS=ON',
+             '-DWITH_SERVER=ON',
+             '-DWITH_CLIENT=ON',
+             '-DWITH_GSSAPI=ON',
+             '-GUnix Makefiles',
+             '-DWITH_PULSE=ON',
+             '-DWITH_CUPS=ON',
+             '-DWITH_SOXR=ON',
+             '-DWITH_LAME=ON',
+             '-DWITH_JPEG=ON',
+             '..'
+      system 'make'
+    end
   end
 
   def self.install
-    system 'make', 'install', "DESTDIR=#{CREW_DEST_DIR}"
+    Dir.chdir('build') do
+      system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    end
   end
 
   def self.postinstall

